docs/reference/ois/next/specification.md

```diff
@@ -375,25 +379,16 @@ node.
         }
       }
     ],
-    "preProcessingSpecifications": [
-      {
-        "environment": "Node",
-        "value": "const output = {...input, from: \"eth\"};",
-        "timeoutMs": "5000"
-      },
-      {
-        "environment": "Node",
-        "value": "const output = {...input, from: input.from.toUpperCase()};",
-        "timeoutMs": "5000"
-      }
-    ],
-    "postProcessingSpecifications": [
-      {
-        "environment": "Node",
-        "value": "const output = Math.round(input.price * 1000);",
-        "timeoutMs": "5000"
-      }
-    ]
+    "preProcessingSpecificationV2": {
```

Comment:
I changed the example here to make it more obvious how v2 is used, but I didn't migrate all of the snippets in the docs (because v1 still works).